package.json

```diff
@@ -24,7 +24,7 @@
     "chart.js": "^4.4.3",
     "classnames": "^2.5.1",
     "copy-webpack-plugin": "^13.0.1",
-    "core-js": "^3.37.1",
+    "core-js": "^3.45.1",
```

Comment:
I think we could even go up to 3.46.0 :) For more context see [here](https://github.com/digitalfabrik/integreat-cms/pull/3953)